templates/zerodb/zerodb.py

```diff
@@ -31,6 +31,16 @@ def _deploy(self):
         zerodb_sal.deploy()
         self.data['ztIdentity'] = zerodb_sal.zt_identity
 
+    def _monitor_disk(self):
+        self.logger.info('Monitor zerodb disk %s' % self.name)
+        disk = self._node_sal.client.system(
```

Comment:
we should really not add new code that use the `system` method. cause this method will be removed in production.